pyo3-build-config/src/impl_.rs

```diff
@@ -101,6 +101,18 @@ pub struct InterpreterConfig {
     /// Serialized to `build_flags`.
     pub build_flags: BuildFlags,
 
+    /// Whether to suppress emitting of `cargo:rustc-link-*` lines from the build script.
+    ///
+    /// Typically, `pyo3`'s build script will emit `cargo:rustc-link-lib=` and
+    /// `cargo:rustc-link-search=` lines derived from other fields in this struct. In
+    /// advanced building configurations, the default logic to derive these lines may not
+    /// be sufficient. This field can be set to `Some(true)` to suppress the emission
+    /// of these lines.
+    ///
+    /// If suppression is enabled, `extra_build_script_lines` should contain equivalent
+    /// functionality or else a build failure is likely.
+    pub suppress_build_script_link_lines: Option<bool>,
```

Comment:
Does it need to be `Option<bool>`? Seems like it could just default to `false` if not present.

Reply:
Fixed in latest push.